### ISSUE TITLE
React hot loading

### ISSUE THREAD

**Issue by ktmn** [fortruce/relay-skeleton#1]
When I edit the main component Application.js here's what happens in console:

```
[WDS] App updated. Recompiling...
client?843a:65 [WDS] App hot update...
only-dev-server.js:69 [HMR] Checking for updates on the server...
log-apply-result.js:11 [HMR] The following modules couldn't be hot updated: (They would need a full reload!)
log-apply-result.js:13 [HMR]  - 480
log-apply-result.js:18 [HMR] Nothing hot updated.
only-dev-server.js:55 [HMR] App is up to date.
```

So basically, it doesn't hot-load the changes. Is this normal?

**Comment by fortruce** [comment]
Hey, thanks for the issue. I ran into this myself, and I just resolved the issue.

The problem is that `react-hot-loader` needs components to be exported and require'd in so that it can be instrumented for hot reloading. It can be fixed by moving `Application` into a separate file `./components/Application.js` and then requiring it in `./containers/Application.js` to wrap it in a Relay Container. Then, pass `./containers/Application` into the `Relay.RootContainer`. All changes to `./components/Application` will then be hot reloaded for you.

I'll be pushing this change up soon.
